applications/sasquatch/charts/app-metrics/values.yaml

```diff
@@ -0,0 +1,114 @@
+## Default values.yaml for the Metrics Events subchart.
+
+# -- app-metrics configuration in any environment in which the subchart is
+# enabled. This should stay globally specified here, and it shouldn't be
+# overridden. Looks like this:
+#
+# globalAppConfig:
+#   some-app:
+#     influxTags:
+#       - some
+#       - tags
+#   some-other-app:
+#     influxTags:
+#       - some
+#       - tags
+#       - and
+#       - others
+globalAppConfig: {}
+
+# -- A list of applications that will publish metrics events, and the keys that should be ingested into InfluxDB as tags.
+# The names should be the same as the app names in Phalanx.
+apps: []
+
+# -- Keys in an every event sent by any app that should be recorded in InfluxDB
+# as "tags" (vs. "fields").
+#
+# -- These will be concatenated with the `influxTags` from `globalAppConfig`
+globalInfluxTags: ["service"]
+
+cluster:
+  # The name of the Strimzi cluster. Synchronize this with the cluster name in
+  # the parent Sasquatch chart.
+  name: sasquatch
+
+# These values refer to the telegraf deployment and config
+
+image:
+  # -- Telegraf image repository
+  repo: "docker.io/library/telegraf"
+
+  # -- Telegraf image tag
+  tag: "1.30.2-alpine"
+
+  # -- Image pull policy
+  pullPolicy: "Always"
+
+# -- Annotations for telegraf-kafka-consumers pods
+podAnnotations: {}
+
+# -- Labels for telegraf-kafka-consumer pods
+podLabels: {}
+
+# -- Secret names to use for Docker pulls
+imagePullSecrets: []
+
+# -- Arguments passed to the Telegraf agent containers
+args: []
+
+# -- Telegraf agent enviroment variables
+# @default -- See `values.yaml`
+env:
+  - name: TELEGRAF_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: sasquatch
+        # Telegraf KafkaUser password.
+        key: telegraf-password
+  - name: INFLUXDB_USER
+    valueFrom:
+      secretKeyRef:
+        name: sasquatch
+        # InfluxDB v1 user
+        key: influxdb-user
+  - name: INFLUXDB_PASSWORD
+    valueFrom:
+      secretKeyRef:
+        name: sasquatch
+        # InfluxDB v1 password
+        key: influxdb-password
+
+# -- Name of the secret with values to be added to the environment.
+envFromSecret: ""
+
+# -- Run Telegraf in debug mode.
+# @default -- false
+debug: false
+
+influxdb:
+  # -- URL of the InfluxDB v1 instance to write to
+  url: "http://sasquatch-influxdb.sasquatch:8086"
+
+# -- Number of Telegraf  replicas. Increase this value to increase the consumer
+# throughput.
+replicaCount: 1
```

Comment:
Just notice that this will increase the consumer throughput only if you create the kafka topics with multiple partitions, where the number of partitions is a multiple of the number of Telegraf replicas.
Even for the EFD use case where we deal with high volumes of data I don't need this.
I think what would be more interesting for the app metrics case is to run 3 replicas for high availability.